getting-started/deployment_guide.md

```diff
@@ -36,42 +38,179 @@ KubeArmor Policy Manager Deployment ...
 KubeArmor Host Policy Manager Service ...
 KubeArmor Host Policy Manager Deployment ...
 ```
+
 </details>
 
-It is assumed that the k8s cluster is already present/reachable setup with the [*required prerequisites*](#Prerequisites) and the user has rights to create service-accounts and cluster-role-bindings.
+It is assumed that the k8s cluster is already present/reachable setup with the [_required prerequisites_](#Prerequisites) and the user has rights to create service-accounts and cluster-role-bindings.
+
+### 3. Deploying sample app
+
+#### a. Deploy sample [wordpress-mysql app](../examples/wordpress-mysql.md)
 
-### 3. Deploying sample app and policies
-   
-#### a. Deploy sample [multiubuntu app](../examples/multiubuntu.md)
 ```
-kubectl apply -f https://raw.githubusercontent.com/kubearmor/KubeArmor/main/examples/multiubuntu/multiubuntu-deployment.yaml
+$ kubectl apply -f https://raw.githubusercontent.com/kubearmor/KubeArmor/main/examples/wordpress-mysql/wordpress-mysql-deployment.yaml
 ```
 
-#### b. Deploy [sample policies](security_policy_examples.md)
+### 4. Use case and deploying Security Policy
```

Comment:
Please use collapsible markdown sections to create usecases and karmor log details below.